pkg/persistence/object_store.go

```diff
@@ -288,6 +306,40 @@ func (s *objectBackupStore) GetBackupVolumeSnapshots(name string) ([]*volume.Sna
 	return volumeSnapshots, nil
 }
 
+func (s *objectBackupStore) GetPodVolumeBackups(name string) ([]*velerov1api.PodVolumeBackup, error) {
+	key := s.layout.getPodVolumeBackupsKey(name)
```

Comment:
It would be nice to extract 1+ helpers from this, since most of this new function is a copy-paste of `GetBackupVolumeSnapshots`. You could have just one that takes the backup name, key, and an interface{} to json-decode into and does the whole process, or you could have one for trying to get the object and another for gzip/json-decoding. If you want to file a tech debt issue to follow up on this, I'm fine with not doing it right now in the interest of getting this merged

Reply:
+1 to merging with a todo/tech debt issue.